client/html/templates/catalog/lists/items-body-list.php

```diff
@@ -176,7 +176,7 @@
 										required="required" <?= !$productItem->isAvailable() ? 'disabled' : '' ?>
 										name="<?= $enc->attr( $this->formparam( array( 'b_prod', 0, 'quantity' ) ) ) ?>"
 									><!--
-									--><button class="btn btn-primary" type="submit" value="">
+									--><button class="btn btn-primary btn-action" type="submit" value="" <?= !$productItem->isAvailable() ? 'disabled' : '' ?>>
```

Comment:
i noticed <kbd>$productItem->isAvailable()</kbd> does not function properly.


![image](https://user-images.githubusercontent.com/82187386/120071951-f8c8d880-c0c3-11eb-9150-f158d2ed9466.png)

Even the stock quantity is zero, it still return true